### PR TITLE
Prometheus NATS exporter

### PR DESCRIPTION
**Pull request by okgolove**
#### What this PR does / why we need it: 

- useful exporter for prometheus for NATS users

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by okgolove** [comment]
Anyone?
@mgoodness @cpanato @prydonius

File: incubator/prometheus-nats-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-nats-exporter.fullname" . }}
+  labels:
+    app: {{ include "prometheus-nats-exporter.name" . }}
```
Comment (cpanato) [inline incubator/prometheus-nats-exporter/templates/deployment.yaml]: lets use the new labels, see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by cpanato** [comment]
also move to stable and set the chart version to 1.0.0

File: incubator/prometheus-nats-exporter/templates/deployment.yaml (hunk 1 above)
Comment (okgolove) [reply incubator/prometheus-nats-exporter/templates/deployment.yaml]: Fixed.

**Comment by okgolove** [comment]
I've moved chart to stable, fixed labels and set chart version to 1.0.0.

**Comment by okgolove** [comment]
Forgot to mention you
@cpanato

**Comment by cpanato** [comment]
```
Error: release prometheus-nats-exporter-ovw3k61722 failed: Deployment.apps "prometheus-nats-exporter-ovw3k61722" is invalid: spec.template.metadata.labels: Invalid value: map[string]string{"app":"prometheus-nats-exporter", "release":"prometheus-nats-exporter-ovw3k61722"}: `selector` does not match template `labels`
```

**Comment by okgolove** [comment]
@cpanato yeah, my fault. Fixed, tested. The chart installs and runs properly with default settings.

**Review by cpanato — APPROVED** [review]

**Comment by okgolove** [comment]
@cpanato sorry, do I need something else to merge this PR?

**Pull request merged**